client/source/DelphiLint.ToolFrame.pas

```diff
@@ -712,9 +717,24 @@ procedure TLintToolFrame.RuleBrowserCreateWebViewCompleted(Sender: TCustomEdgeBr
   DelphiColor: Integer;
 begin
   if not RuleBrowser.WebViewCreated then begin
+    Log.Warn('Rule browser could not be initialized - result code %x', [AResult]);
+    TaskMessageDlg(
+      'DelphiLint''s embedded Edge view could not be initialized.',
+      Format(
+        '(Error code %x)'#13#10'Please ensure that Microsoft Edge 79.0.309 or above is installed and ' +
+        'WebView2Loader.dll is on PATH.',
```

Comment:
You handle this yourself now (previous commit), so there's nothing the user can really do about whether the dll is on PATH.
Alternatively, you could tell them to make sure the dll exists at `%APPDATA%/DelphiLint/bin`?

Reply:
![image](https://github.com/integrated-application-development/delphilint/assets/28214458/c7dda564-2d06-40a2-8ac9-857b89b5f3f0)

How's this?